readme.md

```diff
@@ -0,0 +1,7 @@
+Александр Сычев 953504
+
+Minesweeper is single-player logic-based computer game played on rectangular board whose object is to locate a predetermined number of randomly-placed "mines" in the shortest possible time by clicking on "safe" squares while avoiding the squares with mines. My project includes 3 types of levels (easy, medium, hard) with timer and state with won times
+
+https://www.figma.com/file/8YyG2KucuKViAzZMDMVunX/Untitled?node-id=0%3A1 - mock ups
```

Comment:
Pls, add html tags on top of ur mockups, or in comments. 

I'll check ur pr for the 3rd lab after I'll approve this one. So pls, after approval request my review again, so it won't lost.

Reply:
named elements according to its tags

Reply:
But i provided explaining of what functions do, it's after the '--' sign after the function code, should i remove func code and leave only its name and explanation?

Reply:
I need u to write for example - "Ability to filter by several params" - Cuz ur code, it's a piece of "Ability to filter by several params" in the future I'll ask u to make it, I won't ask u to make field in js, I will ask u to create filtering component with pure js, html, css. That's why I need to know functionality of ur app. Functionality is not equal to "js function", It's equal to feature of ur app